.github/workflows/publish_release.yml

```diff
@@ -28,19 +28,27 @@ jobs:
     if: github.repository == 'microsoft/playwright'
     steps:
     - uses: actions/checkout@v2
+    - uses: azure/docker-login@v1
+      with:
+        login-server: playwright.azurecr.io
+        username: playwright
+        password: ${{ secrets.DOCKER_PASSWORD }}
     - uses: actions/setup-node@v1
       with:
         node-version: 10.15
     - run: npm ci
     - run: npm run build
-    - run: ./docs/docker/build.sh --prepare-context
-    - uses: docker/build-push-action@v1
-      with:
-        username: playwright
-        password: ${{ secrets.DOCKER_PASSWORD }}
-        registry: playwright.azurecr.io
-        repository: public/playwright
-        path: docs/docker/
-        dockerfile: docs/docker/Dockerfile.bionic
-        tags: bionic,latest
-        tag_with_ref: true
+    - name: docker - build & publish
+      run: |
+        # build docker image
+        ./docs/docker/build.sh
+        # GITHUB_REF has a form of `refs/tags/v1.3.0`.
+        # TAG_NAME would be `v1.3.0`
+        export TAG_NAME=${GITHUB_REF#refs/tags/}
+        docker tag playwright:localbuild playwright.azurecr.io/public/playwright:latest
+        docker tag playwright:localbuild playwright.azurecr.io/public/playwright:bionic
+        docker tag playwright:localbuild playwright.azurecr.io/public/playwright:${TAG_NAME}
+        docker rmi playwright:localbuild
+        # push image to registry
+        docker push playwright.azurecr.io/public/playwright:latest
```

Comment:
Since we are only tagging the latest/bionic/v1.30 and not pushing all of them it will result that only latest (`playwright.azurecr.io/public/playwright:latest`) gets updated on the Docker registry. Is this intended?

(You would need to add two more docker push calls then for the other tags)

Reply:
Ah, that's right! I think we can push the repo itself and it'll push all the tags. Let's try on the dev release first.